spec/spec_helper.rb

```diff
@@ -10,7 +10,8 @@
     ActiveRecord::Base.establish_connection(
       adapter: 'postgresql',
       host: 'localhost',
-      username: 'dan',
+      username: ENV["DB_USER"] || nil,
```

Comment:
Sticking with the Postgres env var names would probably cause less confusion.

Reply:
Will fix.